python_ta/__init__.py

```diff
@@ -531,3 +581,61 @@ def doc(msg_id: str) -> None:
     msg_url = HELP_URL + "#" + msg_id.lower()
     print("Opening {} in a browser.".format(msg_url))
     webbrowser.open(msg_url)
+
+
+def _watch_files(
+    file_paths: set,
+    level: str,
+    local_config: Union[dict[str, Any], str],
+    load_default_config: bool,
+    autoformat: Optional[bool],
+    linter: PyLinter,
+    current_reporter: BaseReporter | MultiReporter,
+):
+    """Watch a list of files for modifications and trigger a callback when changes occur."""
+
+    class FileChangeHandler(FileSystemEventHandler):
```

Comment:
Don't make this an "internal" class. It can be defined separately from the `_watch_files` function.